README.md

```diff
@@ -1939,6 +1947,69 @@ This helper for constructing derived channels returns a [*channel*, *setChannel*
 
 Plot.channel is typically used by options transforms to define new channels; these channels are populated (derived) when the custom *transform* function is invoked.
 
+## Scale-aware transforms
+
+Some transforms need to operate in representation space (such as pixels and colors, *i.e.* after scales have been applied) rather than data space. Such a transform might, for example, modify the marks’ positions in screen space to avoid occlusion. These scale-aware transforms are applied *after* the initial setting of the scales, and can modify the channels or derive new channels—which can in turn be passed to scales.
+
+### Dodge
+
+The dodge transform can be applied to any mark that consumes *x* or *y*, such as the Dot, Image, Text and Vector marks.
+#### Plot.dodgeY([*layoutOptions*, ]*options*)
+
+```js
+Plot.dodgeY({x: "date"})
+```
+
+If the marks are arranged along the *x* axis, the dodgeY transform piles them vertically, keeping their *x* position unchanged, and creating a *y* position that avoids overlapping.
```

Comment:
@Fil hi, well, i really wants this functionary. which topic branch is the most updated one (ideally based on plot v0.4.3)?

Reply:
specifically, i want these `Plot.link`s to be stacked (currently overlapping):
![image](https://user-images.githubusercontent.com/117807/166623309-ee571869-04ef-4a27-ab72-c9efdfc17e27.png)

work-arounds like `opacity` and dashed lines isn't satisfactory... xD

and it seems i can specify the stacking direction with `anchor` with `top` & `bottom`? perfect. :)

Reply:
@ryoqun the current branch is https://github.com/observablehq/plot/tree/mbostock/reinitialize; please feel free to test it and give us feedback. (But note that it's highly probable that more changes will occur before this ships.)